js_modules/dagster-ui/packages/ui-core/src/instance/backfill/BackfillPage.tsx

```diff
@@ -140,15 +146,25 @@ export const BackfillPage = () => {
       return runsPathWithFilters(filters);
     }
 
-    const linkQuery = () => {
-      if (backfill.assetBackfillData?.rootAssetTargetedRanges?.length === 1) {
-        const ranges = backfill.assetBackfillData?.rootAssetTargetedRanges;
-        if (ranges?.length) {
-          const {start, end} = ranges[0]!;
-          return {default_range: `[${start}...${end}]`};
-        }
+    const onShowAssetDetails = async (assetKey: AssetKey) => {
+      const resp = await client.query<
```

Comment:
rather than using the client why not `useLazyQuery` ?

Reply:
Hmm - looked at some callsites and I think we mostly useLazyQuery in places where we want the result of the query stored in state + rendered... In this case we want something strictly imperative since we just use the result and redirect rather than rendering it. So i think this still makes sense...